pkg/cmd/server/api/validation/node.go

```diff
@@ -113,3 +115,14 @@ func ValidateDockerConfig(config api.DockerConfig, fldPath *field.Path) field.Er
 func ValidateKubeletExtendedArguments(config api.ExtendedArguments, fldPath *field.Path) field.ErrorList {
 	return ValidateExtendedArguments(config, kubeletoptions.NewKubeletServer().AddFlags, fldPath)
 }
+
+func ValidateVolumeConfig(config api.VolumeConfig, fldPath *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+
+	fsGroupLocalQuotaPath := fldPath.Child("localQuota").Child("fsGroup")
```

Comment:
`fldPath.Child("localQuota", "fsGroup")`.  Also, why not embed it right where you use it below?